### PR TITLE
[SQL] Enable running BeamSqlLine from gradle

### PR DESCRIPTION
**Pull request by akedin**
Allows running SQL shell using gradle command:

```sh
$ ~/w/beam> ./gradlew :beam-sdks-java-extensions-sql-shell:run --console plain -q --no-daemon
applyPortabilityNature with default configuration for project beam-model-pipeline
applyPortabilityNature with default configuration for project beam-model-job-management
applyPortabilityNature with default configuration for project beam-model-fn-execution
Welcome to Beam SQL 2.7.0-SNAPSHOT (based on sqlline version 1.4.0)
0: BeamSQL> SELECT 'asdasd';
+--------+
| EXPR$0 |
+--------+
| asdasd |
+--------+
1 row selected (0.995 seconds)
0: BeamSQL>
```

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | --- | ---

### CODE REVIEW

**Comment by akedin** [comment]
R: @apilloud @amaliujia

File: sdks/java/extensions/sql/shell/build.gradle [hunk 1]
```diff
@@ -37,3 +37,7 @@ processResources {
 }
 
 mainClassName = "org.apache.beam.sdk.extensions.sql.jdbc.BeamSqlLine"
+
```
Comment (amaliujia) [inline sdks/java/extensions/sql/shell/build.gradle]: Can we add comment `./gradlew :beam-sdks-java-extensions-sql-shell:run --console plain -q --no-daemon` as an example of how to run BeamSqlLine from gradle?

**Comment by amaliujia** [comment]
LGTM

File: sdks/java/extensions/sql/shell/build.gradle (hunk 1 above)
Comment (akedin) [reply sdks/java/extensions/sql/shell/build.gradle]: Great suggestion. Updated.

**Pull request merged**